floor/run-show.py

```diff
@@ -7,34 +7,38 @@
 from floor.controller.midi import MidiManager
 from floor.server.server import run_server
 from floor.processor import all_processors
-from floor.util.simple_profile import profile
 
 import argparse
 import importlib
 import os
 import sys
 import logging
+import re
 
 LOG_FORMAT = '%(asctime)-15s | %(name)-12s (%(levelname)s): %(message)s'
 
-SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
-CONFIG_DIR = os.path.join(SCRIPT_DIR, 'config')
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+CONFIG_DIR = os.path.join(BASE_DIR, 'config')
 PLAYLIST_DIR = os.path.join(CONFIG_DIR, 'playlists')
 MIDI_MAPPING_DIR = os.path.join(CONFIG_DIR, 'midi_maps')
+
 DEFAULT_PLAYLIST = os.path.join(PLAYLIST_DIR, 'default.json')
-DEFAULT_FLOOR_CONFIG_FILE = os.path.join(SCRIPT_DIR, 'config/floor-layout.json')
 
 logger = logging.getLogger('show')
 
 
+def driver_to_classname(name):
+    return re.sub(r'(?:^|_)([a-z])', lambda x: x.group(1).upper(), name)
```

Comment:
I wanted to be able to convert some_package_name to SomePackageName

Reply:
Another thing we could do, assuming there are rarely going to be more than a small handful of drivers, is just have a table.

```py
# Maps symbolic driver name to the module implementing it.
ALL_DRIVERS = {
  'raspberry': floor.driver.rapsberry,
  'devserver': floor.driver.devserver,
}
```

I think it's slightly more preferable, from a "what's going on here?" readability standpoint, to a dynamic approach; but this isn't a major concern.

Reply:
Hmm, yah, that's probably better.